src/baseclasses/helper/archive_builder/gamry_archive.py

```diff
@@ -171,11 +171,22 @@ def get_cc_properties(metadata):
 
 def get_atmosphere_data(metadata):
     properties = Atmosphere()
-    properties.ambient_pressure = metadata.get('AIRPRESSURE')
+    try:
```

Comment:
should here be `float`around?

Reply:
It works for decimals with . so I guess we do not need float here and it directly takes float values with the get?
I use float in the except block because there I have to convert to str before.

Reply:
I checked: `type(metadata.get('AIRTEMPERATURE'))` gives `<class 'float'>` even for an example file with `AIRTEMPERATURE	 LABEL	25`
So we do not need float around.

Reply:
thanks for testing then i will merge